test/e2e/framework/framework.go

```diff
@@ -272,30 +270,26 @@ func (f *frameworkWrapper) EnsureTestFederatedNamespace(allClusters bool) *unstr
 		}
 	}
 
-	err = dynclient.Create(context.Background(), obj)
+	err = client.Create(context.Background(), obj)
 	if err != nil {
-		tl.Fatalf("Error creating %s %q: %v", apiResource.Kind, key, err)
+		tl.Fatalf("Error creating %s %q: %v", apiResource.Kind, namespace, err)
```

Comment:
A federated namespace is namespaced, so as repetitive as it may be the name will be `<namespace>/<namespace>`, not `<namespace>`.  If you don't like that, rewording as follows would work: 

`Error creating  <kind> for namespace <namespace>`

Reply:
Done. used the latter. Thanks !